src/jit/codegencommon.cpp

```diff
@@ -5228,6 +5228,13 @@ void CodeGen::genFnPrologCalleeRegArgs(regNumber xtraReg, bool* pXtraRegClobbere
 
                 emitAttr size = emitActualTypeSize(destMemType);
 
+#if defined(_TARGET_ARM64_) && FEATURE_SIMD
+                if (EA_SIZE(size) > 8)
```

Comment:
I wonder: why is this logic arm64/simd specific? Does x64/x86 just handle EA_16BYTE properly in emitIns_R_R, or not see it? Do you really need the `&& defined(FEATURE_SIMD)`?

Reply:
The `&& defined(FEATURE_SIMD)` should not be needed.

I am not sure why it works on XARCH.  It might have something to do with differences in the calling convention for XARCH.  For ARM64 these are passed in integer registers.  Not sure how they are passed for XARCH.

Reply:
Looks for XARCH, like `emitIns_R_R` handles any size < 32 bytes.